modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -2988,7 +2989,7 @@ private double computeBaseOffset(int index) {
         return baseOffset;
     }
 
-    private int computeCurrentIndex() {
+    protected int computeCurrentIndex() {
```

Comment:
This would make this part of the exposed public API. It looks like you are just using this for testing, so making it package scope is sufficient. Can you also add a comment that this is used for testing?